test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -263,25 +264,64 @@ private static void getFailureReason() {
      * testWindow.
      *
      * @param testWindow test window that the test is created
-     * @param position  position can be either HORIZONTAL (both test
-     *                  instruction frame and test window as arranged
-     *                  side by side) or VERTICAL (both test instruction
-     *                  frame and test window as arranged up and down)
+     * @param position  position can either be:
+     *                  HORIZONTAL - both test instruction frame and test window
+     *                  as arranged side by side.
+     *                  VERTICAL - both test instruction frame and test window
+     *                  as arranged up and down.
+     *                  TOP_LEFT_CORNER - test instruction frame positioned at
+     *                  top left corner with main test window beside it.
```

Comment:
All of these descriptions are lacking in explaining what goes where
eg
HORIZONTAL should say the instruction window will be placed so that the right edge is to the left of the screen's horizontal centre and the test window will be placed just to the right of it.

TOP_LEFT_CORNER should say the instruction window will be placed so that its top left corner is at the top left corner of the screen and the test window will be placed just to the right of the instruction window.

Reply:
@prrace  I have slightly changed the description as per your suggestion. Please let me know if any changes are required.

<img width="1720" alt="Horizontal" src="https://user-images.githubusercontent.com/95945681/180101017-43b7aa84-0a0e-4413-9846-c23d27d2798b.png">

**HORIZONTAL - the test instruction frame is positioned such that the right edge aligns with screen's vertical center and the test window is placed to the right of the instruction frame.**

---------------------------------------------------------------------------------------------------
<img width="1720" alt="vertical" src="https://user-images.githubusercontent.com/95945681/180101260-02679903-1d7b-405b-b687-1bdb3b5a75dc.png">

**VERTICAL - the test instruction frame is positioned such that the bottom edge aligns with the screen's horizontal center and the test window is placed below the instruction frame.**

----------------------------------------------------------------------------------------------------
![TopLeft](https://user-images.githubusercontent.com/95945681/180102430-2723a306-ec93-4801-b768-50302a1e4dda.jpg)

**TOP_LEFT_CORNER - the test instruction frame is positioned such that its top left corner is at the top left corner of the screen and the test window is placed to the right of the instruction frame. (Taskbar placed at the top of the screen in this example)**